packages/medusa/src/services/__mocks__/swap.js

```diff
@@ -11,8 +11,17 @@ export const SwapServiceMock = {
     return Promise.resolve()
   }),
   retrieve: jest.fn().mockImplementation(data => {
-    return Promise.resolve({ id: "test-swap" })
+    switch (data) {
```

Comment:
This change, while here happening in the mock itself, is occurring a lot in this PR; replace returns with switches - sometimes on the data directly, sometimes using the query (i.e. when mocking a `findOne` / `findOneWithRelations` function)